cmd/core-data/main.go

```diff
@@ -54,6 +54,7 @@ func main() {
 		data.Configuration,
 		startupTimer,
 		[]interfaces.BootstrapHandler{
+			handlers.SecretClientBootstrapHandler,
```

Comment:
Is there a command-line switch (or other mechanism) that would preclude the need to execute this handler?

Reply:
Good point. I believe there is, or going to be, a configuration property which will determine if we are using a secret store.I have to look into this.

Reply:
@michaelestrin , do you know where the configuration property that would drive this be located?

Reply:
Security is on by default. It can be turned off by running the config-seed with the appropriate environment variable overrides (see PR https://github.com/edgexfoundry/edgex-go/pull/1756). In order to check this value from its location in Consul, a second RegistryClient will be needed.

If no registry is being used, a service can also be invoked with the following env var to disable security `EDGEX_SECURITY_SECRET_STORE=false`. See notes from 15-Aug Core WG. Same is utilized by the docker-edgex-mongo init process.

https://github.com/edgexfoundry/docker-edgex-mongo/blob/0769d9f0c8416f69ea421b9bf45ba0e6c6735a44/internal/pkg/constants.go#L20